packages/driver/cypress/integration/e2e/uncaught_errors_spec.js

```diff
@@ -83,6 +83,20 @@ describe('uncaught errors', () => {
     cy.get('.trigger-async-error').click()
   })
 
+  // we used to wrap timers with "proxy" tracking functions
+  // this has been called from the top frame
+  // and thus its error handler has been catching the error and not the one in AUT
```

Comment:
@chrisbreiding I'm curious - how things got replaced in your PR so you don't require timers wrapping anymore? your PR is quite lengthy and I didn't have that much time to dig into it, if you could share a little bit more details for my curious mind I would highly appreciate it 😅

Reply:
I moved the timer-wrapping into the code we inject into the AUT, so that errors thrown from timer callbacks get caught by the AUT error handler and not the top frame's error handler. Here's the entry point for the injection code: https://github.com/cypress-io/cypress/blob/7.0-release/packages/runner/injection/index.js

Reply:
Cool! 

I'm working on the iframe-based app so I'm wondering if there is a simple way to get access to the `timers.wrap` - I guess for the best experience we should call it within our iframe's context as well. Note that we already have a code responsible for injecting code into our iframe asap so that part is not a problem. If there is not - would you consider exposing this somehow?

PS. just a small protip which I've discovered and can't imagine living without now. Press `y` before copying the GitHub URL to a file which will change the location address for the one containing the commit hash 🎉

Reply:
The timer wrapping is only used for the sake of screenshots, so that we can pause timers before screenshots in an attempt to make screenshots between runs consistent. If that's not important for your app, it might not be necessary to go to the trouble.

Better iframe support is in the pipeline and is actually one of my next priorities. With that, we'll be automatically injecting stuff like the timer wrapping into nested iframes, so it won't be necessary to manually do it.

Reply:
I don't actually remember if we are using screenshots when rendering in the iframe or are we just rendering a "full screen" app when we need this. Either way - if the iframe support is in the pipeline then definitely we don't need to bother with this for now. 

I'm also super pumped about this - we have done some quirky stuff to inject stuff into our iframe. It got better when `cy.route2` has been released but there are still some other things that we had to do. I've actually started to write a blog post about this - might need to throw it in the dumpster if this knowledge will become useless soon-ish 😜